script.py

```diff
@@ -4,12 +4,12 @@
 import base64
 import collections
 import datetime
-import json
 import os
 import re
-import requests
 import subprocess
 
+import requests
+
```

Comment:
I installed `requests` so long ago, I forgot it wasn't a builtin! Thanks for rearranging these.